src/GitHub.TeamFoundation.14/Services/VSGitServices.cs

```diff
@@ -60,20 +60,31 @@ public string GetLocalClonePathFromGitProvider()
         }
 
         /// <inheritdoc/>
-        public async Task Clone(
-            string cloneUrl,
-            string clonePath,
-            bool recurseSubmodules,
-            object progress = null)
+        public async Task Clone(string cloneUrl, string clonePath, bool recurseSubmodules, object progress = null)
+        {
+#if (!TEAMEXPLORER14)
```

Comment:
Indeed, we can get rid of all `#if` and consolidate to a single project. I just wanted a commit where it worked but hardly touched the `GitHub.TeamFoundation.*` projects.

I'll now strip out the unnecessary stuff. 😄